src/promptflow/promptflow/_sdk/_orm/run_info.py

```diff
@@ -49,10 +49,12 @@ class RunInfo(Base):
     end_time = Column(TEXT)  # ISO8601("YYYY-MM-DD HH:MM:SS.SSS"), string
     data = Column(TEXT)  # local path of original run data, string
     run_source = Column(TEXT)  # run source, string
+    is_eager_mode = Column(Boolean, default=False)  # mark if the run is an eager flow run
```

Comment:
when VSCode Extension use this?

Reply:
When visualizing? I think it's better to persist this instead of check by file extension + content each time.

Reply:
I think they just need to know whether this flow has nodes or not? no need a special bool field?

Reply:
And more basic question is this eager_mode thing is a flow property, why it need to go to run obj?

Reply:
Offline synced, will avoid run interface change for eager flow